File: content/en/api/service_level_objectives/code_snippets/api-slo-delete-many.py
from datadog import initialize, api

options = {
    'api_key': '<DATADOG_API_KEY>',
    'app_key': '<DATADOG_APPLICATION_KEY>'
}

slo_ids = ["<YOUR_SLO_ID>", "<YOUR_SLO_ID>"]

initialize(**options)

api.ServiceLevelObjective.delete_many(slo_ids)

File: content/en/api/service_level_objectives/code_snippets/api-slo-delete-many.rb
require 'dogapi'

api_key = '<YOUR_API_KEY>'
app_key = '<YOUR_APP_KEY>'
slo_ids = ['<YOUR_SLO_ID>', '<YOUR_SLO_ID>']

dog = Dogapi::Client.new(api_key, app_key)

# Delete multiple SLOs
dog.delete_many_service_level_objective(slo_ids)
